react-frontend-app/src/App.js

```diff
@@ -20,25 +20,61 @@ import BarGraph from './components/BarGraph'
 function App() {
   const [surname, setSurname] = useState('');
   const [zipcode, setZip] = useState('');
-  var prediction = {};
+
+  // ZRP inputs
+  const [firstName, setFirst] = useState('');
+  const [midName, setMid] = useState('');
+  const [precinct, setPrecinct] = useState('');
+  const [gender, setGender] = useState('');
+  const [county, setCounty] = useState('');
+  const [cong, setCong] = useState('');
+  const [senate, setSenate] = useState('');
+  const [house, setHouse] = useState('');
+  const [birth, setBirth] = useState('');
 
   // Functions that are called whenever the text input is changed
   function handleSurname(event) {
     setSurname(event.target.value);
   }
-
   function handleZip(event) {
     setZip(event.target.value);
   }
+  function handlefirstName(event) {
+    setFirst(event.target.value);
+  }
+  function handlemidName(event) {
+    setMid(event.target.value);
+  }
+  function handlePrecinct(event) {
+    setPrecinct(event.target.value);
+  }
+  function handleGender(event) {
+    setGender(event.target.value);
+  }
+  function handleCounty(event) {
+    setCounty(event.target.value);
+  }
+  function handleCong(event) {
+    setCong(event.target.value);
+  }
+  function handleSenate(event) {
+    setSenate(event.target.value);
+  }
+  function handleHouse(event) {
+    setHouse(event.target.value);
+  }
+  function handleBirth(event) {
+    setBirth(event.target.value);
+  }
 
   // Function that called when the submit button is pressed
   function handleSubmit(event) {
     event.preventDefault();
     console.log('Submit was pressed');
```

Comment:
Eventually, we'll want to remove these console.log statements — you can do it now or later on!